scenes/Restaurant/RestaurantScene.gd

```diff
@@ -0,0 +1,27 @@
+extends Node2D
+
+var total_shake_seconds := 0.0
+var remaining_shake_seconds := 0.0
+var shake_magnitude := 2.5
```

Comment:
Consider naming these fields `shake_remaining_seconds`, `shake_total_seconds` and `shake_original_position` to more clearly group them.

Reply:
Renamed as suggested